client/app/locales/fr.coffee

```diff
@@ -42,6 +42,8 @@ module.exports =
     'vendorLink': "Site d'où sont récupérées les données : "
     'loginUrl': "URL d'authentification"
     'token': "Token"
+    'refreshToken': "Mettre à jour le token"
+
```

Comment:
We should remove one of the blanklines, it would be easier to diff two language files. But that can be done after the merge